prime-router/metadata/organizations-local.yml

```diff
@@ -13,11 +13,30 @@
   - name: az-phd
     description: Arizona PHD
     services:
-      - name: elr
+      - name: elr-test
```

Comment:
In all the organizations file, there are now two services: elr-test and elr-prod, for AZ.  @mry-usds ,can you eyeball this?

Reply:
Looks good -- @cglodosky - this will be a change to the passwords/usernames (i.e should be AZ_PHD__ELR_TEST__USER)